.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @JourneyBook
```

Comment:
```suggestion
* @collab-community/journey-book 
```
This may work not sure.

Reply:
even what i did may not work, this is better, merging but we gotta test.